frontend/src/components/common/header/index.css

```diff
@@ -150,3 +150,40 @@ a.dropdown-item {
     display: none;
   }
 }
+
+.resource-heading {
+  margin-top: 60px;
+  margin-bottom: 60px;
+  font-size: 48px;
+}
+
+.resource-title {
+  margin-top: 60px;
+  margin-top: 60px;
+}
+
+.resource-p {
+  margin-bottom: 60px;
+  font-size: 24px;
+}
+
+.resource-p2 {
+  font-size: 20px;
+}
+
+.resource-list {
+  color: black !important;
+  font-size: 24px;
+}
+
+.test {
```

Comment:
lets come up with a better name for this -> maybe just call it `lightgray`

and then do width on the jumbo tron like so: 

```
Jumbotron {
  width: 100%
}

.lightgray {  background-color: lightgray }
```

i dont think the fixed height is needed at this point, creates a lot of white space?

Reply:
I believe the blue links have been fixed by @ktburns214 in a different PR. As far as the jumbotron taking up 100vh, I kinda wanted to show as a proof of concept of the hash-link behavior combined with 100vh.